### DISCUSSION TITLE
Thoughts on the legality of hacking games~

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#591] [(A) Anything]
The majority of video game hacking developers have been prosecuted for copyright violations to get rid of them easily, but all Prodigy did is DMCA our extension, because we didn't really use their assets in this repository. However, I have a few concerns:

1. Hacking free membership could be considered piracy/theft if Prodigy had a good attorney.
2. Developing extensions, apps, etc. could be a copyright violation depending on how we use Prodigy content.
3. That whole idea we had for an employee dashboard hack or whatever was illegal in more ways than one and isn't a good idea at all.

Therefore, we need to be _very_ careful about how we distribute hacks as to not violate any laws; it's in a very grey area.

**Comment by PatheticMustan** [comment]
1. I'm not really concerned with the piracy/theft thing, since it's a very big stretch. It's also super expensive to hire an attorney. 
2. Yeah, this is probably why we got DMCA'd. Next time, we won't include screenshots of Prodigy on our extension page, or their assets ;)
3. Yeah, not gonna pursue that any further.

**Reply by ghost** [reply]
> I'm not really concerned with the piracy/theft thing, since it's a very big stretch. It's also super expensive to hire an attorney.

Prodigy probably has an attorney considering how large they are. A few hack developers got sued for hacking people in-game items that would otherwise cost money.

**Reply by ghost** [reply]
[Oh, yes, they have a legal counsel.](https://github.com/github/dmca/blob/1501ce56ba992fe1d9a6d9f93992c0de78512269/2020/04/2020-04-21-Prodigy.md)

**Comment by irhrhd** [comment]
Why are they targeting craftersshaft his old prodigy hacking repo was the
one i was in

**Reply by ghost** [reply]
They targeted Prodigy Reborn because it was the easiest to get rid of.

**Comment by irhrhd** [comment]
Wow so f**k**g rude

**Reply by ghost** [reply]
No, it's called capitalism. Silencing anybody that opposes your big corporation and getting a crap ton of money. Capitalism.

**Comment by irhrhd** [comment]
Yeah i wish capitalizm was a person that i culd just kill to get rid of

**Reply by ghost** [reply]
Well, at least it's not communism.

**Comment by irhrhd** [comment]
Okay dont get me started about communism
